cluster/addons/metrics-server/metrics-server-deployment.yaml

```diff
@@ -81,15 +81,16 @@ spec:
         command:
           - /pod_nanny
           - --config-dir=/etc/config
-          - --cpu=40m
+          - --cpu={{ base_metrics_server_cpu }}
           - --extra-cpu=0.5m
-          - --memory=40Mi
-          - --extra-memory=4Mi
+          - --memory={{ base_metrics_server_memory }}
```

Comment:
Inconsistency: lack of Mi here, Mi below.

Reply:
The implementation of this change is based on how the heapster yaml has been templatized and is consistent with that implementation. I too noticed this inconsistency, but didn't want to make things inconsistent with existing code. I can implement a follow-up PR to clean this up.